packages/components/src/unit-control/types.ts

```diff
@@ -42,7 +41,7 @@ export type UnitControlOnChangeCallback = InputChangeCallback< {
 	data?: WPUnitControlUnit;
 } >;
 
-export type UnitSelectControlProps = Pick< InputControlProps, 'size' > & {
```

Comment:
UnitSelectControl technically only supports two size variants (default and small), so I streamlined this a bit to not accept an `__unstable-large` value.